src/common/pure/InlineBanner/index.tsx

```diff
@@ -57,6 +57,7 @@ const Wrapper = styled.span<{ color: string }>`
     width: var(--size);
     height: var(--size);
     object-fit: contain;
+    stroke: none !important;
```

Comment:
I will tell @fairlighteth  :P

Reply:
It's because of legacy styles in Swap confirm modal...